### PR TITLE
Slider v8: update to support ariaLabel

### LINKED ISSUE #23757: [Bug]: Slider does not have accessible name
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 17.22 GB / 31.81 GB
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (103.0.1264.37), ChromiumDev (104.0.1293.1)
    Internet Explorer: 11.0.22000.120
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://fabricweb.z5.web.core.windows.net/oufr/8.77.2/#/examples/slider

## Actual Behavior
This is a failing test reported by the axe-core devtools (the jest extension) showing the accessibility bug

image

## Expected Behavior
Slider should have an accessible name

### Logs

_No response_

### Requested priority

High

### Products/sites affected

Mission Control

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by smhigley**
Resolves #23757, [15029](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/15029)

We have `ariaLabel` documented as a supported prop, but the current v8 Slider does not support it -- this PR adds support, updates our example to add a label for the first basic Slider, and maintains support for `aria-label`.

### CODE REVIEW

File: packages/react/src/components/Slider/Slider.test.tsx [hunk 1]
```diff
@@ -42,6 +42,16 @@ describe('Slider', () => {
     expect(getByRole('slider').getAttribute('aria-labelledby')).toBe('custom-label');
   });
 
+  it('can set an aria-label attribute', () => {
+    const { getByRole } = render(<Slider aria-label="test" />);
+    expect(getByRole('slider').getAttribute('aria-label')).toBe('test');
+  });
+
+  it('can sets ariaLabel as an aria-label attribute', () => {
```
Comment (sopranopillow) [inline packages/react/src/components/Slider/Slider.test.tsx:50]: typo:
```suggestion
  it('can set ariaLabel as an aria-label attribute', () => {
```

**Review by sopranopillow — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/40e67af4ad7af8b0881976c94a28e3110abc0548'>40e67af4ad7af8b0881976c94a28e3110abc0548</a></sub>
<div id="bundleSizeComment9423"/>

**Review by spmonahan — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/BaseButton-mount_base.html">788</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/BaseButton-mount.html">785</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Breadcrumb-mount_base.html">2285</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Breadcrumb-mount.html">2261</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Checkbox-mount_base.html">1271</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Checkbox-mount.html">1271</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/CheckboxBase-mount_base.html">1062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/CheckboxBase-mount.html">1070</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ChoiceGroup-mount_base.html">3913</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ChoiceGroup-mount.html">3946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ComboBox-mount_base.html">837</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ComboBox-mount.html">779</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/CommandBar-mount_base.html">undefined</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/CommandBar-mount.err.txt">err</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ContextualMenu-mount_base.html">9252</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ContextualMenu-mount.html">9241</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DefaultButton-mount_base.html">956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DefaultButton-mount.html">961</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DetailsRow-mount_base.html">3051</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DetailsRow-mount.html">3061</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DetailsRowFast-mount_base.html">3066</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DetailsRowFast-mount.html">3066</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DetailsRowNoStyles-mount_base.html">2880</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DetailsRowNoStyles-mount.html">2889</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Dialog-mount_base.html">2312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Dialog-mount.html">2283</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DocumentCardTitle-mount_base.html">152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/DocumentCardTitle-mount.html">153</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Dropdown-mount_base.html">2826</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Dropdown-mount.html">2834</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/FocusTrapZone-mount_base.html">1580</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/FocusTrapZone-mount.html">1553</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/FocusZone-mount_base.html">1552</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/FocusZone-mount.html">1527</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/IconButton-mount_base.html">1412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/IconButton-mount.html">1391</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Label-mount_base.html">316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Label-mount.html">318</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Layer-mount_base.html">2535</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Layer-mount.html">2571</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Link-mount_base.html">441</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Link-mount.html">422</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/MenuButton-mount_base.html">1206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/MenuButton-mount.html">1202</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/MessageBar-mount_base.html">1840</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/MessageBar-mount.html">1819</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Nav-mount_base.html">2649</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Nav-mount.html">2870</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/OverflowSet-mount_base.html">1003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/OverflowSet-mount.html">983</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Panel-mount_base.html">1811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Panel-mount.html">1805</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Persona-mount_base.html">858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Persona-mount.html">838</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Pivot-mount_base.html">1137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Pivot-mount.html">1127</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/PrimaryButton-mount_base.html">1084</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/PrimaryButton-mount.html">1081</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Rating-mount_base.html">6726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Rating-mount.html">6711</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SearchBox-mount_base.html">1117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SearchBox-mount.html">1117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Shimmer-mount_base.html">2211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Shimmer-mount.html">2235</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Slider-mount_base.html">1707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Slider-mount.html">1672</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SpinButton-mount_base.html">3964</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SpinButton-mount.html">3928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Spinner-mount_base.html">393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Spinner-mount.html">404</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SplitButton-mount_base.html">2461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SplitButton-mount.html">2477</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Stack-mount_base.html">493</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Stack-mount.html">477</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/StackWithIntrinsicChildren-mount_base.html">1910</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/StackWithIntrinsicChildren-mount.html">1876</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/StackWithTextChildren-mount_base.html">4680</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/StackWithTextChildren-mount.html">4662</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SwatchColorPicker-mount_base.html">9054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/SwatchColorPicker-mount.html">9020</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/TagPicker-mount_base.html">2033</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/TagPicker-mount.html">2049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/TeachingBubble-mount_base.html">70679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/TeachingBubble-mount.html">69823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Text-mount_base.html">397</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Text-mount.html">397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/TextField-mount_base.html">1191</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/TextField-mount.html">1154</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ThemeProvider-mount_base.html">950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ThemeProvider-mount.html">945</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ThemeProvider-virtual-rerender_base.html">605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ThemeProvider-virtual-rerender.html">589</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1442</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Toggle-mount_base.html">722</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/Toggle-mount.html">712</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/buttonNative-mount_base.html">105</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23776/perf-test/buttonNative-mount.html">109</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by khmakoto — APPROVED** [review]

**Pull request merged**